components/ILIAS/Contact/PRIVACY.md

```diff
@@ -0,0 +1,87 @@
+- The presentation includes a presentation of the user's profile (restricted depending on the user's profile publishing
+  settings) and control elements presenting the user's options for contact requests ("Confirm Contact"/"Ignore Contact").
+- Initially the options are restricted to requesting a contact.
+- The requesting user can cancel the request.
+- The requested user has the option to actively ignore or confirm the request.
+- Both contacts can unlink the contact without any further consent.
+- Each party can detect a change of the contact relation within its own contact options. The only exception to this
+  is the active ignoring of a request. The requesting user can not detect if his request was ignored.
+- A list of all contacts is shown as a selection when a user adds members to a newly created mailing list.
+- The "Look up Users" functionality within the composition of a mail can provide a list of contacts if such matches with
+  the search term exist.
+- Furthermore, a list of user accounts with membership assignment is shown by clicking the "List Members" action for an
+  objects within the "My Courses/My Groups" view, accessible directly from the "Compose" view of "Mail"
+  component, or by clicking the "Contacts" link item in the main bar. For each account, the username is presented.
+- Additionally, full names are presented according to the "Profile and Privacy > Publish Profile > Visible for logged-in
+  Users" setting. The "Contact Status" is shown as well.
+- The "Mail to Members" view of membership-enabled object types, which is also located in the "Contact" component,
+  provides a list of participating user accounts. For each account, the username is presented and additionally, the full
+  name, if published.
+  - This information does not show any information about contact relations.
+- Contacts are referenced within the mail service in the sub tab "Contacts".
+- A contact relation cannot be perceived within the user interface by any user other than the contacts themselves.
+
+## Data being deleted
+
+- If a contact request is canceled, the stored entry is removed without any traceable footprint.
+- If a contact request is ignored, the flagged entry in the database will be preserved to prevent a repeated contact
+  request from the same initiator.
+- If a contact is unlinked by one of the contacts, the stored entry is removed without any traceable footprint.
+- Any user relations that originate from the contact are not affected by its deletion. This includes preservation in
+  mailing lists and participation within chatrooms.
+- If a user is deleted, its contacts, send and received contact requests, mailing lists and mailing list participations
+  are removed without any traceable footprint.
+
+## Data being exported
+
+- The component does not provide any export option itself.
+- A contact request can trigger a notification, which may be displayed as a "Toast" to the requested user. The 
```

Comment:
I have no problem with Toast, but I have found that less experienced users get confused by it. I think it might be useful to add a short description or rephrase a sentence, maybe smth like: 
"A contact request can trigger a notification, which may be displayed as a "Toast" to the requested user. A "Toast" is a notification element that contains the user presentation name (depending on his visibility) and the user intention of the contact request." 
or (more detailed): 
"A contact request can trigger a notification, which may be displayed as a "Toast" to the requested user. A "Toast" is a notification element that is displayed in the top right-hand corner of the screen and contains the user presentation name (depending on his visibility) and the user intention of the contact request."  - But I'm perfectly fine with "Toast is a notification element".

Reply:
I choose the short one because I tried to balance the description here. I agree that this should be more explanatory, but at the same time this part should not set properties of the toast and therefore imply a restriction on it. (e.g. "in the top right-hand corner of the screen" is a toast property and should be defined by toast and not the contact, especially since this is not relevant for the contact-concerning part here)